docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -31,6 +31,7 @@ acorn install
       --cluster-domain strings                          The externally addressable cluster domain (default .oss-acorn.io)
       --controller-replicas int                         acorn-controller deployment replica count
       --controller-service-account-annotation strings   annotation to apply to the acorn-system service account
+      --event-ttl string                                Amount of time an Acorn event will be stored before being deleted (default '24h' - 24 hours)
```

Comment:
can we do seven days or will that generate too much data?

Reply:
done